src/components/SplitVertical/SplitVertical.spec.jsx

```diff
@@ -17,38 +18,49 @@ describe('SplitVertical', () => {
 				<SplitVertical />
 			);
 
+			const motionWrapper = wrapper.find(Motion).shallow();
+
 			assert.equal(wrapper.find('.lucid-SplitVertical').length, 1);
-			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner').length, 1);
-			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner > .lucid-SplitVertical-LeftPane').length, 1);
-			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner > .lucid-SplitVertical-Divider').length, 1);
-			assert.equal(wrapper.find('.lucid-SplitVertical > .lucid-SplitVertical-inner > .lucid-SplitVertical-RightPane').length, 1);
+			assert.equal(motionWrapper.find('.lucid-SplitVertical-inner').length, 1);
+			assert.equal(motionWrapper.find('.lucid-SplitVertical-inner > .lucid-SplitVertical-LeftPane').length, 1);
+			assert.equal(motionWrapper.find('.lucid-SplitVertical-inner > .lucid-SplitVertical-Divider').length, 1);
+			assert.equal(motionWrapper.find('.lucid-SplitVertical-inner > .lucid-SplitVertical-RightPane').length, 1);
 		});
 	});
 
 	describe('props', () => {
-		describe('isExpanded', () => {
+		describeWithDOM('isExpanded', () => {
+			let mountWrapper;
+
+			afterEach(() => {
+				if (mountWrapper) {
+					mountWrapper.unmount();
+				}
+			});
+
 			it('should default to true', () => {
 				const wrapper = shallow(
 					<SplitVertical />
 				);
 
-				assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-expanded').length, 1);
+				assert(wrapper.hasClass('lucid-SplitVertical-is-expanded'));
 			});
 
 			it('should apply the &-is-expanded css class when true', () => {
 				const wrapper = shallow(
 					<SplitVertical isExpanded={true} />
 				);
 
-				assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-expanded').length, 1);
+				assert(wrapper.hasClass('lucid-SplitVertical-is-expanded'));
 			});
 
-			it('should not apply the &-is-expanded css class when false', () => {
-				const wrapper = shallow(
+			it('should not apply the &-is-expanded css class when false [mostly stable]', (done) => {
+				mountWrapper = mount(
 					<SplitVertical isExpanded={false} />
 				);
 
-				assert.equal(wrapper.find('.lucid-SplitVertical.lucid-SplitVertical-is-expanded').length, 0);
+				assert(!mountWrapper.hasClass('lucid-SplitHorizontal-is-expanded'));
+				_.delay(done, 10);
```

Comment:
this was really weird: because if i didn't do that, i got an error saying that i was trying to access an unmounted elemented.